tests/framework/Optimizers/SimulatedAnnealing/BoltzmanSA/gold/opt_export.xml

```diff
@@ -0,0 +1,10 @@
+<DataObjectMetadata name="opt_export">
```

Comment:
I'm guessing you don't mean to check the XML in all tests, right? I don't think there's anything interesting to check except for maybe on one of the test cases, as there's no unique information in here.

Reply:
OK